src/consensus/validation.h

```diff
@@ -54,6 +54,8 @@ enum class TxValidationResult {
     TX_CONFLICT,
     TX_MEMPOOL_POLICY,        //!< violated mempool's fee/size/descendant/RBF/etc limits
     TX_NO_MEMPOOL,            //!< this node does not have a mempool so can't validate the transaction
+    TX_SINGLE_FAILURE,        //!< fee was insufficient to meet some policy, but can change if submitted in a (different) package
+    TX_UNKNOWN,               //!< transaction was not validated
```

Comment:
As with this PR, the only code place where we fulfilled a `TX_UNKNOWN` as `TxValidationResult` is within `AcceptPackage()`, if we don’t find wtxid in `individual_results_nonfinal`. AFAICT, this can happen when we have caught errors in subpackage and we break early from linearized package validation. If so, I think comment can be extended “transaction was not validated, early error its in package was caught” (could be called `TX_PACKAGE_FAILURE`)

Reply:
Didn't rename because I don't like having the word "failure" in it (this tx didn't fail!). But added a "because package failed" to the comment.